scripts/install/jellyfin.sh

```diff
@@ -108,219 +76,56 @@ CONFIG
-      </DisabledMetadataFetchers>
-      <MetadataFetcherOrder />
-      <DisabledImageFetchers />
-      <ImageFetcherOrder />
-    </MetadataOptions>
-    <MetadataOptions>
-      <ItemType>Episode</ItemType>
-      <DisabledMetadataSavers />
-      <LocalMetadataReaderOrder />
-      <DisabledMetadataFetchers>
-        <string>The Open Movie Database</string>
-        <string>TheMovieDb</string>
-      </DisabledMetadataFetchers>
-      <MetadataFetcherOrder />
-      <DisabledImageFetchers>
-        <string>The Open Movie Database</string>
-        <string>TheMovieDb</string>
-      </DisabledImageFetchers>
-      <ImageFetcherOrder />
-    </MetadataOptions>
-  </MetadataOptions>
-  <EnableAutomaticRestart>true</EnableAutomaticRestart>
-  <SkipDeserializationForBasicTypes>false</SkipDeserializationForBasicTypes>
   <BaseUrl />
-  <UICulture>en-US</UICulture>
-  <SaveMetadataHidden>false</SaveMetadataHidden>
-  <ContentTypes />
-  <RemoteClientBitrateLimit>0</RemoteClientBitrateLimit>
-  <EnableFolderView>false</EnableFolderView>
-  <EnableGroupingIntoCollections>false</EnableGroupingIntoCollections>
-  <DisplaySpecialsWithinSeasons>true</DisplaySpecialsWithinSeasons>
-  <LocalNetworkSubnets />
   <LocalNetworkAddresses>
     <string>0.0.0.0</string>
   </LocalNetworkAddresses>
-  <CodecsUsed />
-  <IgnoreVirtualInterfaces>false</IgnoreVirtualInterfaces>
-  <EnableExternalContentInSuggestions>true</EnableExternalContentInSuggestions>
-  <RequireHttps>false</RequireHttps>
-  <IsBehindProxy>false</IsBehindProxy>
-  <EnableNewOmdbSupport>false</EnableNewOmdbSupport>
-  <RemoteIPFilter />
-  <IsRemoteIPFilterBlacklist>false</IsRemoteIPFilterBlacklist>
-  <ImageExtractionTimeoutMs>0</ImageExtractionTimeoutMs>
-  <PathSubstitutions />
-  <EnableSimpleArtistDetection>true</EnableSimpleArtistDetection>
-  <UninstalledPlugins />
 </ServerConfiguration>
 CONFIG
 #
-# Create the service file that will start and stop jellyfin.
-cat > "/etc/systemd/system/jellyfin.service" <<-SERVICE
-[Unit]
-Description=Jellyfin
-After=network.target
-
-[Service]
-User=${username}
-Group=${username}
-UMask=002
-
-Type=simple
-WorkingDirectory=$install_dir
-ExecStart=$install_dir/jellyfin -d /home/${username}/.config/Jellyfin
-TimeoutStopSec=20
-KillMode=process
-Restart=always
-RestartSec=2
-
-[Install]
-WantedBy=multi-user.target
-SERVICE
-#
 # Configure the nginx proxypass using positional parameters.
 if [[ -f /install/.nginx.lock ]]; then
     bash "/usr/local/bin/swizzin/nginx/jellyfin.sh" "${app_port_http}" "${app_port_https}"
     systemctl reload nginx
 fi
 #
 # Set the correct and required permissions of any directories we created or modified.
-chown "${username}.${username}" -R "$install_dir"
-chown "${username}.${username}" -R "$install_ffmpeg"
 chown "${username}.${username}" -R "/home/${username}/.config"
 chown "${username}.${username}" -R "/home/${username}/.ssl"
+chmod -R g+rw "/home/${username}/.config/Jellyfin"
+chmod -R g+r "/home/${username}/.ssl"
```

Comment:
where is this dir made?

Reply:
1: https://github.com/liaralabs/swizzin/pull/424/files#diff-1dd9197f2356f16752770c87f5fca467a14da6824ef3da7d92515209de38f17fR41

2: by the ssl function

Reply:
/resolved